src/components/plots/plots.module.css

```diff
@@ -11,4 +11,5 @@
     font-weight: 400;
     bottom: 10px;
     transform-origin: 0% 0%;
-}
\ No newline at end of file
+    left: -3px;
```

Comment:
I just noticed the text for this wasn't right up against the edge of the component box like it was in the design